src/utils/notifications.ts

```diff
@@ -117,6 +129,28 @@ export function clearAllNotifications(client: MatrixClient): Promise<Array<{} |
     return Promise.all(receiptPromises);
 }
 
+/**
+ * Gives the marked_unread state of the given room
+ * @param room The room to check
+ * @returns - The marked_unread state of the room, or undefined if no explicit state is set.
+ */
+export function getMarkedUnreadState(room: Room): boolean | undefined {
+    const currentStateStable = room.getAccountData(MARKED_UNREAD_TYPE_STABLE)?.getContent<IMarkedUnreadEvent>()?.unread;
+    const currentStateUnstable = room
+        .getAccountData(MARKED_UNREAD_TYPE_UNSTABLE)
+        ?.getContent<IMarkedUnreadEvent>()?.unread;
+    return currentStateStable ?? currentStateUnstable;
+}
+
+export async function setUnreadMarker(room: Room, client: MatrixClient, unread: boolean): Promise<void> {
```

Comment:
document please

Reply:
why is it called "setUnreadMarker", when the getter is called "getMarkedUnreadState"? Shouldn't it be called "setMarkedUnreadState" or possibly "setMarkedUnread"?